ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -65,8 +67,23 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
 
     // Check for unwanted data; these messages aren't thrown out, but this class of errors will be
     // ignored for most pipeline monitoring.
-    if ("com-turkcell-yaani".equals(namespace)) {
-      throw new UnwantedDataException("1612933");
+    HashMap<String, String> ignoredNamespaces = new HashMap<String, String>() {
+
+      {
+        put("com-turkcell-yaani", "1612933");
+        put("org-mozilla-fenix-beta", "1612934");
+        put("org-mozilla-vrbrowser-dev", "1614410");
+        put("org-mozilla-fenix-performancetest", "1614412");
+      }
```

Comment:
This feels a bit more efficient rather than having `if` statements for each unwanted namespace.

Reply:
fancy. I haven't seen this method of defining a map before. I like it.

These variables should be moved outside the function, so they aren't recreated for each message.